heliopy/data/ulysses.py

```diff
@@ -70,7 +73,23 @@ def swics_heavy_ions(starttime, endtime, try_download=True):
     for ion in ['ALPHA', 'C6', 'O6', 'NE8', 'MG10', 'SI9', 'SI10', 'FE11']:
         names += ['DENS_' + ion, 'VEL_' + ion, 'TEMP_' + ion]
     product = 'uswimatb'
-    return _swics(starttime, endtime, names, product, try_download)
+    units = OrderedDict([('VEL_ALPHA', u.km / u.s), ('TEMP_ALPHA', u.K),
+                        ('VEL_C6', u.km / u.s), ('TEMP_C6', u.K),
+                        ('VEL_O6', u.km / u.s), ('TEMP_O6', u.K),
+                        ('VEL_NE8', u.km / u.s), ('TEMP_NE8', u.K),
+                        ('VEL_MG10', u.km / u.s), ('TEMP_MG10', u.K),
+                        ('VEL_SI9', u.km / u.s), ('TEMP_SI9', u.K),
+                        ('VEL_SI10', u.km / u.s), ('TEMP_SI10', u.K),
+                        ('VEL_FE11', u.km / u.s), ('TEMP_FE11', u.K),
+                        ('DENS_O6', u.cm**-3), ('DENS_ALPHA', u.dimensionless_unscaled),
```

Comment:
I feel like all the other `"DENS_` columns should have units, do you have a link to the documentation where you got the other units from?

Reply:
In the docstrings, it states that other densities are the ratio of that ion to the Oxygen 6+ ion, so I did that, I will take a look and see if I can find the source

Reply:
Here! http://ufa.esac.esa.int/ufa-sl-server/data-action?PROTOCOL=HTTP&PRODUCT_TYPE=ALL&FILE_NAME=readme.txt&FILE_PATH=/ufa/HiRes/data/swics

Reply:
👍